examples/mediadevices/public/snippet.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const {
+  __createLocalAudioTrack__,
+  __createLocalVideoTrack__,
+  __LocalAudioTrack__,
+  __LocalVideoTrack__
```

Comment:
-1—I don't think the hyperlink functionality is worth the `__` prefix and suffix. Let's kick out to a subsequent PR?